docs/docs/APIdocs/layers/nn/Initializers/ConstInitMethod.md

```diff
@@ -0,0 +1,71 @@
+## ConstInitMethod ##
+
+
+**Scala:**
+``` scala
+val initMethod = ConstInitMethod(value: Double)
+
+```
+**Python:**
+```python
+init_method = ConstInitMethod(value, bigdl_type="float")
+```
+
+Initialization method that set tensor to the specified constant value.
+
+
+**Scala example:**
+```scala
+import com.intel.analytics.bigdl.nn._
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric.NumericFloat
+
+
+val weightInitMethod = ConstInitMethod(0.2)
+val biasInitMethod = ConstInitMethod(0.2)
+val model = Linear(3, 2).setName("linear1")
+model.setInitMethod(weightInitMethod, biasInitMethod)
```

Comment:
A little bit confusing here. You defined a layer but name it a model (usually models are defined as containers Sequential, etc.).  How about we write it this way: 
```scala
val layer = Linear(3, 2).setName("linear1")
layer.setInitMethod(weightInitMethod=ConstInitMethod(0.2), biasInitMethod=ConstInitMethod(0.2))
val model = Sequential()
model.add(layer)
println(model.getParametersTable().get("linear1").get)
```